bower.json

```diff
@@ -2,7 +2,7 @@
   "name": "convert-ember-components",
   "dependencies": {
     "handlebars": "~1.3.0",
-    "jquery": "^1.11.1",
+    "jquery": "^2.1.0",
```

Comment:
It might actually be better to specify `> 1.9.0` since there's a legitimate reason to use 1.9.x (IE8 support).